tfx_bsl/beam/run_inference.py

```diff
@@ -427,7 +427,7 @@ def _sending_as_binary(input_name: Text) -> bool:
 
   @staticmethod
   def _parse_feature_content(values: Sequence[Any], attr_name: Text,
-                             as_binary: bool) -> Sequence[Any]:
+                             as_binary: bool) -> List[Any]:
```

Comment:
I think you can still use Sequence here. Sequence means the returned list is immutable.

Reply:
This change was in response to Sheryl's comment above. Currently, this function may return `google.protobuf.pyext._message.RepeatedScalarContainer` which is a `Sequence` but not a `List` and it causes errors later on with json serialization. This PR performs conversion to `List` which ensures it will be json serializable